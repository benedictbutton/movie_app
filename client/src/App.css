.App {
  background-color: #0b0b0b;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
  font-size: calc(10px + 2vmin);
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
}

.App-header {
  width: 100%;
  /* color: white; */
  border-bottom-style: solid;
  border-color: lightgrey;
}

.App-link {
  color: #61dafb;
}

.home {
  background-image: url("./assets/grey-large.jpg");
  background-repeat: no-repeat;
  -moz-background-size: 100% 100%;
  -webkit-background-size: 100% 100%;
  background-size: 100% 100%;
  flex: 1;
  width: 100%;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
}

.theater {
  background-image: url("./assets/theater.jpg");
  background-repeat: no-repeat;
  -moz-background-size: 100% 100%;
  -webkit-background-size: 100% 100%;
  background-size: 100% 100%;
  display: flex;
  flex: 1;
  width: 100%;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
  /* direction: row;
  position: relative; */
}

.sticky-container {
  width: 100%;
  position: relative;
}

.sticky-boundary {
  position: relative;
  margin: 0;
  min-height: 120vh;
}

/* .scrollbar::-webkit-scrollbar {
  overflow-y: scroll;
  height: 12px;
  width: 12px;
  background: yellow;
}

.scrollbar::-webkit-scrollbar-thumb {
  background: black;
  -webkit-border-radius: 1ex;
  -webkit-box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.75);
}

.scrollbar::-webkit-scrollbar-corner {
  background: #000;
} */

.sticky-box {
  position: relative;
  position: -webkit-sticky;
  position: sticky;
  z-index: 1;
}

.sticky-box.is-top {
  top: 0px;
}

.sticky-box.is-top.is-sticky {
  box-shadow: 0 7px 2px -2px darkgray;
}

.sticky-box.is-bottom {
  bottom: 0px;
}
.sticky-box.is-bottom.is-sticky {
  box-shadow: 0 -7px 2px -2px darkgray;
}

.u-center-center {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}
.u-center-top {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-start;
}

.u-feature-detect-sticky {
  position: absolute;
  position: sticky;

  position: -webkit-sticky;
  position: -moz-sticky;
  position: -ms-sticky;
  position: -o-sticky;
  position: sticky;
}

#image {
  background-image: url("./assets/grey-small.jpg");
  background-repeat: no-repeat;
  -moz-background-size: 100% 100%;
  -webkit-background-size: 100% 100%;
  background-size: 100% 100%;
}

#rotate {
  /* Safari */
  -webkit-transform: rotate(45deg);
  /* Firefox */
  -moz-transform: rotate(45deg);
  /* IE */
  -ms-transform: rotate(45deg);
  /* Opera */
  -o-transform: rotate(45sdeg);
  /* Internet Explorer */
  filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=3);
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
