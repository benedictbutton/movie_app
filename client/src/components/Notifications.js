import React from "react";
import { connect } from "react-redux";
import { doUnSetError } from "../redux/actions/notificationActions";
//material-ui
import Typography from "@material-ui/core/Typography";
import Modal from "@material-ui/core/Modal";
import { withStyles } from "@material-ui/core/styles";

function rand() {
  return Math.round(Math.random() * 20) - 10;
}

function getModalStyle() {
  const top = 50 + rand();
  const left = 50 + rand();

  return {
    top: `${top}%`,
    left: `${left}%`,
    transform: `translate(-${top}%, -${left}%)`
  };
}

const styles = theme => ({
  paper: {
    position: "absolute",
    width: theme.spacing.unit * 50,
    backgroundColor: theme.palette.background.paper,
    boxShadow: theme.shadows[5],
    padding: theme.spacing.unit * 4
  }
});

const Notifications = props => {
  const { children, classes } = props;

  return (
    <div>
      {children ? (
        <Modal
          aria-labelledby="simple-modal-title"
          aria-describedby="simple-modal-description"
          open={children.display}
          onClose={props.doUnSetError}
        >
          <div style={getModalStyle()} className={classes.paper}>
            <Typography variant="h6" id="modal-title" align="center">
              {children.message}
            </Typography>
            <Typography variant="subtitle1" id="simple-modal-description" />
          </div>
        </Modal>
      ) : null}
    </div>
  );
};

const mapStateToProps = state => ({
  client: state.client
});

export default connect(
  mapStateToProps,
  { doUnSetError }
)(withStyles(styles)(Notifications));
