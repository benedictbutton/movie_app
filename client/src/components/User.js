import React, { useState } from "react";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import backgroundImage from "../assets/dark-grey-metal.jpg";
import userImage from "../assets/no-poster.jpg";
// material-ui
import Button from "@material-ui/core/Button";
import EditIcon from "@material-ui/icons/Edit";
import Fade from "@material-ui/core/Fade";
import Grid from "@material-ui/core/Grid";
import Paper from "@material-ui/core/Paper";
import Snackbar from "@material-ui/core/Snackbar";
import StarIcon from "@material-ui/icons/Star";
import TextField from "@material-ui/core/TextField";
import Tooltip from "@material-ui/core/Tooltip";
import Typography from "@material-ui/core/Typography";
import { withStyles } from "@material-ui/core/styles";

const styles = theme => ({
  root: {
    flex: 1,
    width: "100%",
    height: "100%",
    backgroundColor: "#eeeeee"
  },
  main: {
    flex: 1,
    justifyContent: "space-around",
    alignItems: "flex-start"
  },
  layout: {
    padding: theme.spacing.unit * 3,
    margin: theme.spacing.unit * 3,
    background: `url(${backgroundImage})`
  },
  grid: {
    height: "auto",
    padding: "56.25% 0 0 0",
    maxWidth: 200
  },
  media: {
    width: "75%",
    height: "50%"
  },
  btnMargin: {
    marginTop: theme.spacing.unit
  },
  rightIcon: {
    marginLeft: theme.spacing.unit
  },
  starGrid: {
    flex: 1
  },
  stars: {
    background: "rgba(0, 0, 0, 0.5)",
    maxHeight: 48
  },
  overview: {
    color: "#343434"
  },
  input: {
    color: "white",
    borderColor: "white"
  },
  cssLabel: {
    color: "white"
  },
  cssOutlinedInput: {
    "&$cssFocused $notchedOutline": {
      borderColor: "white !important"
    }
  },
  notchedOutline: {
    borderWidth: "2px",
    borderColor: "white !important"
  },
  cssFocused: { borderWidth: "2px", borderColor: "white" },
  textField: {
    marginLeft: theme.spacing.unit,
    marginRight: theme.spacing.unit
  },

  title: {
    marginBottom: theme.spacing.unit * 3,
    color: "#2F4F4F",
    fontWeight: "700"
  },
  snack: {
    color: "white",
    backgroundColor: theme.palette.primary.dark
  }
});

export const User = props => {
  const { classes } = props;
  const { username, email } = props.user.profile;

  const [open, setOpen] = useState(false);

  const stars = [1, 2, 3, 4, 5].map(el => (
    <StarIcon key={el} size={20} style={{ color: "yellow" }} />
  ));

  return (
    <Grid container className={classes.root}>
      <Grid item xs={12} md={8}>
        <Paper className={classes.layout}>
          <Grid container className={classes.main} spacing={16}>
            <Grid item xs={12} sm={5} className={classes.grid}>
              <img
                src={userImage}
                className={classes.media}
                alt="avatar image"
              />
              <Paper className={classes.stars}>
                <Grid
                  container
                  className={classes.starGrid}
                  alignItems="center"
                  justify="space-evenly"
                >
                  {stars}
                </Grid>
              </Paper>
              <Grid container className={classes.btnMargin}>
                <Grid item xs={12}>
                  <Tooltip title="Clear rating" placement="bottom">
                    <Button
                      variant="outlined"
                      color="primary"
                      alt="clear rating"
                      fullWidth
                      onClick={() => setOpen(true)}
                    >
                      Upload Image
                      <FontAwesomeIcon
                        className={classes.rightIcon}
                        icon="upload"
                      />
                    </Button>
                  </Tooltip>
                </Grid>
              </Grid>
            </Grid>
            <Grid item sm={7}>
              <Grid container>
                <Grid item xs={6}>
                  <TextField
                    id="outlined-read-only-input"
                    label="Username"
                    value={username}
                    className={classes.textField}
                    margin="normal"
                    InputLabelProps={{
                      classes: {
                        root: classes.cssLabel,
                        focused: classes.cssFocused
                      }
                    }}
                    InputProps={{
                      readOnly: true,
                      classes: {
                        input: classes.input,
                        root: classes.cssOutlinedInput,
                        notchedOutline: classes.notchedOutline,
                        focused: classes.cssFocused
                      }
                    }}
                    variant="outlined"
                  />
                </Grid>
                <Grid item xs={6}>
                  <TextField
                    id="outlined-read-only-input"
                    label="Email"
                    value={email}
                    className={classes.textField}
                    margin="normal"
                    InputLabelProps={{
                      classes: {
                        root: classes.cssLabel,
                        focused: classes.cssFocused
                      }
                    }}
                    InputProps={{
                      readOnly: true,
                      classes: {
                        input: classes.input,
                        root: classes.cssOutlinedInput,
                        notchedOutline: classes.notchedOutline,
                        focused: classes.cssFocused
                      }
                    }}
                    variant="outlined"
                  />
                </Grid>
                <Grid item xs={12}>
                  <TextField
                    fullWidth
                    id="outlined-read-only-input"
                    label="Password"
                    defaultValue="**********"
                    className={classes.textField}
                    margin="normal"
                    InputLabelProps={{
                      classes: {
                        root: classes.cssLabel,
                        focused: classes.cssFocused
                      }
                    }}
                    InputProps={{
                      readOnly: true,
                      classes: {
                        input: classes.input,
                        root: classes.cssOutlinedInput,
                        notchedOutline: classes.notchedOutline,
                        focused: classes.cssFocused
                      }
                    }}
                    variant="outlined"
                  />
                </Grid>
                <Grid
                  container
                  className={classes.btnMargin}
                  justify="flex-end"
                >
                  <Grid item xs={4}>
                    <Tooltip
                      title={true ? "Remove from playlist" : "Add to playlist"}
                      placement="bottom"
                    >
                      <Button
                        variant="contained"
                        color="primary"
                        alt="clear rating"
                        fullWidth
                        onClick={() => setOpen(true)}
                      >
                        <EditIcon style={{ color: "white" }} />
                      </Button>
                    </Tooltip>
                  </Grid>
                </Grid>
              </Grid>
            </Grid>
          </Grid>
        </Paper>
      </Grid>{" "}
      <Snackbar
        className={classes.snack}
        anchorOrigin={{ vertical: "top", horizontal: "right" }}
        autoHideDuration={5000}
        open={open}
        onClose={() => setOpen(false)}
        ContentProps={{
          "aria-describedby": "message-id",
          className: classes.snack
        }}
        TransitionComponent={Fade}
        message={<span id="message-id"> TBD - Coming soon</span>}
      />
    </Grid>
  );
};

export default withStyles(styles)(User);
