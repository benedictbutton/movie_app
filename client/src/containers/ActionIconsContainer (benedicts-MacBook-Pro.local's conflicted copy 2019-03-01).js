import React from "react";
import classNames from "classnames";
import Grid from "@material-ui/core/Grid";
import { withStyles } from "@material-ui/core/styles";

export const styles = theme => ({
  /* Styles applied to the root element. */
  grid: {
    position: "absolute",
    left: 0,
    right: 0,
    height: 48
  },
  root: {
    alignItems: "center"
  },
  /* Styles applied to the root element if `titlePosition="bottom"`. */
  titlePositionBottom: {
    background: "rgba(0, 0, 0, 0.5)",
    bottom: 0,
    alignItems: "center",
    justifyContent: "space-between"
  },
  /* Styles applied to the root element if `titlePosition="top"`. */
  titlePositionTop: {
    background: "transparent",
    top: 0,
    alignItems: "center"
  }
});

const ActionIconsContainer = props => {
  const {
    starsVisible,
    actionIcon,
    actionPosition,
    classes,
    className: classNameProp,
    titlePosition
  } = props;

  const className = classNames(classes.grid, {
    [classes.titlePositionBottom]: titlePosition === "bottom",
    [classes.titlePositionTop]: titlePosition === "top"
  });

  return (
    <div>
      {actionIcon && starsVisible ? (
        <Grid container className={classes.root}>
          <Grid item className={className}>
            {actionIcon}
          </Grid>
        </Grid>
      ) : null}
    </div>
  );
};

ActionIconsContainer.defaultProps = {
  actionPosition: "right",
  titlePosition: "bottom"
};

export default withStyles(styles, { name: "MuiGridListTileBar" })(
  ActionIconsContainer
);
