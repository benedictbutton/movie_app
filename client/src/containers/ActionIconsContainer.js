import React from 'react'
import classNames from 'classnames'
import Grid from '@material-ui/core/Grid'
import { withStyles } from '@material-ui/core/styles'

export const styles = (theme) => ({
    /* Styles applied to the root element. */
    root: {
        position: 'absolute',
        left: 0,
        right: 0,
        height: '48',
        paddingBottom: theme.spacing.unit,
    },
    /* Styles applied to the root element if `titlePosition="bottom"`.  */
    titlePositionBottom: {
        background: 'rgba(0, 0, 0, 0.5)',
        bottom: 0,
    },
    /* Styles applied to the root element if `titlePosition="top"`. */
    titlePositionTop: {
        background: 'transparent',
        top: 0,
        justifyContent: 'flex-end',
    },
})

const ActionIconsContainer = (props) => {
    const { starsVisible, actionIcon, classes, titlePosition } = props

    const className = classNames(classes.root, {
        [classes.titlePositionBottom]: titlePosition === 'bottom',
        [classes.titlePositionTop]: titlePosition === 'top',
    })

    return (
        <div>
            {actionIcon && starsVisible ? (
                <Grid container className={className}>
                    {actionIcon}
                </Grid>
            ) : null}
        </div>
    )
}

ActionIconsContainer.defaultProps = {
    actionPosition: 'right',
    titlePosition: 'bottom',
}

export default withStyles(styles, { name: 'MuiGridListTileBar' })(
    ActionIconsContainer
)
