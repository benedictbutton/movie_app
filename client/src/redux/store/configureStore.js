import { createStore, applyMiddleware } from "redux";
import { createLogger } from "redux-logger";
import createSagaMiddleware from "redux-saga";
import rootReducer from "../reducers/index";
import rootSaga from "../sagas";
import { loadState, saveState } from "../../localStorage";
import throttle from "lodash/throttle";

const logger = createLogger();
const saga = createSagaMiddleware();
const persistedState = loadState();

const store = createStore(
  rootReducer,
  persistedState,
  applyMiddleware(saga, logger)
);

saga.run(rootSaga);

store.subscribe(
  throttle(() => {
    saveState({
      user: store.getState().user,
      ratings: store.getState().ratings
    });
  }, 1000)
);

export { store };
