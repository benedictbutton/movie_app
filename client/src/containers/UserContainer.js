import React, { Component } from "react";
import { Redirect, withRouter } from "react-router-dom";
import { connect } from "react-redux";
import { doUserRequesting } from "../redux/actions/userActions";
import User from "../components/User";

class UserContainer extends Component {
  componentDidMount() {
    const { history } = this.props;
    this.props.dispatch(doUserRequesting(history));
  }

  render() {
    return <User {...this.props} />;
  }
}

const mapStateToProps = (state, props) => ({
  client: state.client,
  user: state.user
});

export default withRouter(
  connect(
    mapStateToProps,
    null
  )(UserContainer)
);
