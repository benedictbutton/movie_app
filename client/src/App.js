import React, { Component } from 'react'
import { Route, Switch } from 'react-router-dom'
import './App.css'
//font awesome
import { library } from '@fortawesome/fontawesome-svg-core'
import {
    faSignOutAlt,
    faBars,
    faLock,
    faUser,
    faFilm,
    faStar,
    faStepForward,
    faPlay,
    faUpload,
} from '@fortawesome/free-solid-svg-icons'
//App Components
import Admin from './components/Admin'
import HeaderContainer from './containers/HeaderContainer'
import HomeContainer from './containers/HomeContainer'
import MovieContainer from './containers/MovieContainer'
import MoviesContainer from './containers/MoviesContainer'
import RatingsContainer from './containers/RatingsContainer'
import PlaylistContainer from './containers/PlaylistContainer'
import PlaylistsContainer from './containers/PlaylistsContainer'
import PrivateRoute from './HOC/PrivateRoute'
import ResetPasswordContainer from './containers/ResetPasswordContainer'
import SignInContainer from './containers/SignInContainer'
import SignUpContainer from './containers/SignUpContainer'
import UserContainer from './containers/UserContainer'
import MediaUpdate from './components/MediaUpdate'
import PersonContainer from './containers/PersonContainer'

import Fix from './components/Fix'

library.add(
    faSignOutAlt,
    faBars,
    faUser,
    faLock,
    faFilm,
    faStar,
    faStepForward,
    faPlay,
    faUpload
)

class App extends Component {
    render() {
        return (
            <div className="App">
                <Route exact path="/" component={HomeContainer} />
                <header className="App-header">
                    <Route path="/ms" component={HeaderContainer} />
                </header>
                <Switch>
                    <Route path="/ms/sign-up" component={SignUpContainer} />
                    <Route path="/ms/sign-in" component={SignInContainer} />
                    <Route
                        path="/ms/reset_password/:id/:pass"
                        component={ResetPasswordContainer}
                    />
                    <PrivateRoute path="/ms/admin" component={Admin} />
                    <PrivateRoute
                        path="/ms/movies/:type/:query?/:desc?/:end?"
                        component={MoviesContainer}
                    />
                    <PrivateRoute
                        path="/ms/movie/:id"
                        component={MovieContainer}
                    />
                    <PrivateRoute
                        path="/ms/ratings"
                        component={RatingsContainer}
                    />
                    <PrivateRoute
                        path="/ms/person/:id"
                        component={PersonContainer}
                    />
                    <PrivateRoute
                        path="/ms/playlists"
                        component={PlaylistsContainer}
                    />
                    <PrivateRoute
                        path="/ms/playlist/:id"
                        component={PlaylistContainer}
                    />
                    <PrivateRoute
                        path="/ms/users/:id"
                        component={UserContainer}
                    />
                    <Route path="/ms/media_update" component={MediaUpdate} />
                    <Route path="/ms/fix" component={Fix} />
                </Switch>
            </div>
        )
    }
}

export default App
//
