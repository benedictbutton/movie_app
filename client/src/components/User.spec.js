import React from "react";
import renderer from "react-test-renderer";
import toJson from "enzyme-to-json";
import configureStore from "redux-mock-store";
import createSagaMiddleware from "redux-saga";
import { Provider } from "react-redux";
import { BrowserRouter as Router, MemoryRouter } from "react-router-dom";
import App from "../App";
import Notifications from "./Notifications";
import User from "./User";
import { UserContainer } from "../containers/UserContainer";

const sagaMiddleware = createSagaMiddleware();
const middlewares = [sagaMiddleware];
const mockStore = configureStore(middlewares);
const initialState = {
  client: {
    requesting: false,
    successful: false,
    admin: false,
    reset: false,
    notifications: { message: "test" }
  },
  user: {
    profile: {
      id: 7,
      firstName: "test",
      lastName: "user",
      username: "testuser",
      email: "testuser@email.com"
    },
    requesting: false,
    successful: false,
    notifications: {
      body: null,
      message: "test",
      code: 404,
      display: true
    }
  }
};

// console.log(wrapper.debug());

describe("User container/component", () => {
  let store;
  store = mockStore(initialState);
  beforeEach(() => jest.resetAllMocks());

  const user = {
    profile: {
      id: 7,
      firstName: "test",
      lastName: "user",
      username: "testuser",
      email: "testuser@email.com"
    },
    notifications: {
      body: null,
      message: "test",
      code: 404,
      display: true
    }
  };

  const dispatch = store.dispatch;

  jest.mock("../redux/actions/userActions", () => ({
    doUserRequesting: jest.fn()
  }));

  describe("UserContainer", () => {
    it("render()", () => {
      const wrapper = mount(
        <Provider store={store}>
          <UserContainer user={user} dispatch={dispatch} />
        </Provider>
      );
      expect(wrapper.find(User)).toHaveLength(1);
      const component = wrapper.find(User);
      expect(component.props().user.profile.firstName).toEqual("test");
      wrapper.unmount();
    });
  });

  let wrapper;
  describe("User", () => {
    test("renders consistently", () => {
      wrapper = shallow(<User user={user} />);
      const component = wrapper.dive();
      expect(toJson(component)).toMatchSnapshot();
    });

    it("render()", () => {
      // const wrapper = shallow(<User user={user} />);
      console.log(wrapper.debug());
      expect(wrapper.exists()).toBe(true);
      wrapper.unmount();
    });
  });
});
