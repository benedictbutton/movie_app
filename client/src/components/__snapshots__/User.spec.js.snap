// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User renders consistently 1`] = `
<Fragment>
  <WithStyles(Grid)
    className="User-root-1"
    container={true}
  >
    <LoadingIndicator>
      <Component />
    </LoadingIndicator>
    <div
      position="absolute"
    >
      <pure(AccountCircleIcon)
        className="User-icon-2"
        style={
          Object {
            "fontSize": 200,
          }
        }
      />
      <WithStyles(Grid)
        alignItems="center"
        container={true}
        justify="center"
      >
        <WithStyles(Grid)
          item={true}
          xs={5}
        >
          <WithStyles(Paper)
            align="center"
            className="User-paper-3"
          >
            <WithStyles(Typography)
              className="User-type-4"
              gutterBottom={true}
              variant="h2"
            >
              Profile
            </WithStyles(Typography)>
            <WithStyles(Grid)
              alignItems="flex-start"
              container={true}
              justify="space-around"
              spacing={24}
            >
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-type-4"
                  component="h4"
                  variant="h5"
                >
                  First Name
                </WithStyles(Typography)>
              </WithStyles(Grid)>
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-info-5"
                  component="h3"
                  variant="h5"
                >
                  test
                </WithStyles(Typography)>
              </WithStyles(Grid)>
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-type-4"
                  component="h4"
                  variant="h5"
                >
                  Last Name
                </WithStyles(Typography)>
              </WithStyles(Grid)>
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-info-5"
                  component="h3"
                  variant="h5"
                >
                  user
                </WithStyles(Typography)>
              </WithStyles(Grid)>
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-type-4"
                  component="h4"
                  variant="h5"
                >
                  Username
                </WithStyles(Typography)>
              </WithStyles(Grid)>
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-info-5"
                  component="h3"
                  variant="h5"
                >
                  testuser
                </WithStyles(Typography)>
              </WithStyles(Grid)>
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-type-4"
                  component="h4"
                  variant="h5"
                >
                  Email
                </WithStyles(Typography)>
              </WithStyles(Grid)>
              <WithStyles(Grid)
                align="left"
                item={true}
                xs={5}
              >
                <WithStyles(Typography)
                  className="User-info-5"
                  component="h3"
                  noWrap={true}
                  variant="h5"
                >
                  testuser@email.com
                </WithStyles(Typography)>
              </WithStyles(Grid)>
            </WithStyles(Grid)>
          </WithStyles(Paper)>
        </WithStyles(Grid)>
      </WithStyles(Grid)>
    </div>
  </WithStyles(Grid)>
</Fragment>
`;
